src/utils/staking.js

```diff
@@ -218,7 +218,13 @@ export class Staking {
         } else {
             lockupId = getLockupAccountId(accountId)
         }
-        await (await new nearApiJs.Account(this.wallet.connection, lockupId)).state()
+        try {
```

Comment:
there is `wallet.accountExists` helper BTW, we probably just need to extract it into helper function, this pattern now is used in way too many places :)

Reply:
not sure I want to handle this as part of this PR